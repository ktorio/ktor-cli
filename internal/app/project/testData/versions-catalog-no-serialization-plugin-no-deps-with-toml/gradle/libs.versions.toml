[versions]
kotlin-version = "2.0.21"

[plugins]
kotlin-jvm = { id = "org.jetbrains.kotlin.jvm", version.ref = "kotlin-version" }
