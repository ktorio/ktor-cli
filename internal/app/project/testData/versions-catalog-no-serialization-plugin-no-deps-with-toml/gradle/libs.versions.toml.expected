[versions]
ktor = "3.0.0"
kotlin-version = "2.0.21"

[libraries]
ktor-serialization-kotlinx-json = { module = "io.ktor:ktor-serialization-kotlinx-json", version.ref = "ktor" }

[plugins]
kotlin-jvm = { id = "org.jetbrains.kotlin.jvm", version.ref = "kotlin-version" }
kotlin-serialization = { id = "org.jetbrains.kotlin.plugin.serialization", version.ref = "kotlin-version" }
